ORGANIZATION.md

```diff
@@ -83,6 +83,7 @@ Wait a few days. If there are no sponsors and/or speakers that have stepped up,
   * [ ] "We need sponsors: give back to the community!"
   * [ ] "Provisional date for next meetup, raise hands".
   * [ ] "Raise hands who is looking for work? Stand up who is hiring? Mingle!"
+  * [ ] Record number of attendees, number of RSVPs on meetup.com and weather to github issue for respective month (see #25 for example).
```

Comment:
Please move it to just under `take photo during talk 1`. It's more realistic to do it then instead of in the end when the organizer just wants to be done with it and relax :)

Reply:
That's exactly the place I had it at my first go. Moved it to the end on my second try. Sigh. Now moved.